mango-viz/bdgenomics/mango/pileup/sources.py

```diff
@@ -38,7 +38,7 @@
     GA4GHVariantSource
     GA4GHFeatureSource
 """
-from bdgenomics.mango.io import *
+#from bdgenomics.mango.io import *
```

Comment:
Can you move this to a separate PR?

Reply:
I already made a separate PR that does this. Make sure to rebase and this will go away.